src/backend/test/telemetry_service_impl_test.cpp

```diff
@@ -125,6 +137,20 @@ class TelemetryServiceImplTest : public ::testing::Test {
         const float signal_strength_percent) const;
     std::future<void> subscribeRcStatusAsync(std::vector<RcStatus>& rc_status_events) const;
 
+    void checkSendsActuatorControlTargetEvents(
+        const std::vector<ActuatorControlTarget>& actuator_control_target_events) const;
+    ActuatorControlTarget
+    createActuatorControlTarget(const uint8_t group, const float* controls) const;
+    std::future<void> subscribeActuatorControlTargetAsync(
+        std::vector<ActuatorControlTarget>& actuator_control_target_events) const;
+
+    void checkSendsActuatorOutputStatusEvents(
+        const std::vector<ActuatorOutputStatus>& actuator_output_status_events) const;
+    ActuatorOutputStatus
+    createActuatorOutputStatus(const uint8_t group, const float* controls) const;
```

Comment:
Does it have to be pointer? What about e.g. a `const vector&`?

Reply:
Agree. Will do this

Reply:
Changed createActuatorControlTarget in the same way.